src/components/FlowRunGraph.vue

```diff
@@ -0,0 +1,141 @@
+<template>
+  <div class="flow-run-graph">
+    <template v-if="load">
+      <RunGraph
+        v-model:viewport="viewport"
+        v-model:selected="selected"
+        v-model:fullscreen="fullscreen"
+        :config="config"
+        class="flow-run-graph__graph p-background"
+      />
+    </template>
+    <template v-else>
+      <FlowRunGraphConfirmation @confirm="confirm" />
+    </template>
+  </div>
+</template>
+
+<script lang="ts" setup>
+  import { NodeSelection, RunGraph, RunGraphConfig, ViewportDateRange } from '@prefecthq/graphs'
+  import { useColorTheme } from '@prefecthq/prefect-design'
+  import { computed, ref } from 'vue'
+  import FlowRunGraphConfirmation from '@/components/FlowRunGraphConfirmation.vue'
+  import { useTaskRunsCount } from '@/compositions/useTaskRunsCount'
+  import { useWorkspaceApi } from '@/compositions/useWorkspaceApi'
+  import { FlowRun } from '@/models/FlowRun'
+  import { ServerStateType } from '@/models/StateType'
+
+  const OPT_IN_THRESHOLD = 2000
+
+  const props = defineProps<{
+    flowRun: FlowRun,
+    viewport: ViewportDateRange,
+    fullscreen: boolean,
+    selected: NodeSelection | null,
+  }>()
+
+  const emit = defineEmits<{
+    'update:viewport': [ViewportDateRange | undefined],
+    'update:fullscreen': [boolean],
+    'update:selected': [NodeSelection | null],
+  }>()
+
+  const api = useWorkspaceApi()
+  const { value: colorThemeValue } = useColorTheme()
+  const load = ref(true)
+
+  const viewport = computed({
+    get() {
+      return props.viewport
+    },
+    set(value) {
+      emit('update:viewport', value)
+    },
+  })
+
+  const fullscreen = computed({
+    get() {
+      return props.fullscreen
+    },
+    set(value) {
+      emit('update:fullscreen', value)
+    },
+  })
+
+  const selected = computed({
+    get() {
+      return props.selected
+    },
+    set(value) {
+      emit('update:selected', value)
+    },
+  })
+
+  // these will be replaced with brandon's styles
```

Comment:
These are the correct values though I believe. Safe to clean these up later? The actual implementation is kinda messy IMO because I think it uses the bg from dark mode and the text color from light mode or something like that

Reply:
Yeah we can do that. FYI on the text colors, I believe all state colors work well with white text on them (well enough for the state tags at least, prob not always accessible but for another day).